### PR TITLE
Add minimally working TrackJS service

### PR DESCRIPTION
**Pull request by shakeelmohamed**
We're using https://trackjs.com/ over at [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io).

I just copied over the icons from `custom` for now

### CODE REVIEW

File: lib/trackjs/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = {
+  apiVersion: 1,
+  name: 'TrackJS',
+  parse: function(headers, body) {
+    if(body.data && body.data.errors && body.data.errors.length > 0) {
+      var ret = {
+        message: 'TrackJS | ' + body.data.errors[0].message,
+        icon: 'logo',
+        errorLevel: body.data.errors[0].count > 1 ? 'error' : 'normal'
```
Comment (trevorah) [inline lib/trackjs/index.js]: would you ever get a `body.data.errors[0].count` that's less than 1?

**Comment by trevorah** [comment]
Here's a logo you can use:

logo 2x
logo

I created it from their favicon.

File: lib/trackjs/index.js (hunk 1 above)
Comment (shakeelmohamed) [inline lib/trackjs/index.js]: Unlikely, I could change the condition to be `body.data.errors[0].count === 1 ? 'normal' : 'error'` if that's any better

Comment (trevorah) [inline lib/trackjs/index.js]: Nah, keep it as it is. It just seemed odd to have logic for when TrackJS reports that an error has never happened.

All that needs changing now is the logo...

**Comment by shakeelmohamed** [comment]
Thanks for doing that @trevorah, I added the error icons as well :+1:

**Comment by trevorah** [comment]
Thanks @shakeelmohamed, I'll let you know once this is live.

**Pull request merged**